client/reader/index.js

```diff
@@ -49,7 +49,6 @@ export default function() {
 
 		// Old and incomplete paths that should be redirected to /
 		page( '/read/following', '/' );
-		page( '/read', '/' );
 		page( '/read/blogs', '/' );
 		page( '/read/feeds', '/' );
 		page( '/read/blog', '/' );
```

Comment:
These redirects should now redirect to `/read`. If I understand it correctly, paths like `/read/following` are invalid or legacy. The current code might redirect Reader paths to a path that renders `/home`.

Reply:
Thanks, I've just updated these, too.